### ISSUE TITLE
Metadata icon is dark-grey in dark mode

### ISSUE THREAD

**Issue by jannuary** [rafaelmardojai/share-preview#34]
![image](https://user-images.githubusercontent.com/27908024/166944484-d98f5bfd-0045-4d43-9808-13589fa7a864.png)

(also that text should probably not use .large-title)

**Comment by rafaelmardojai** [comment]
Yeah, just noticed I'm installing the icon in the wrong directory.

> (also that text should probably not use .large-title)

Lol, good catch! I thought I removed all the `.large-title`.
